src/main/java/org/simulator/sbml/SBMLinterpreter.java

```diff
@@ -107,7 +107,7 @@ public SBMLinterpreter(Model model)
   public SBMLinterpreter(Model model, double defaultSpeciesValue, double defaultParameterValue,
       double defaultCompartmentValue)
       throws SBMLException, ModelOverdeterminedException {
-    this(model, 0d, 1d, 1d, null);
```

Comment:
ErrorProne complained about this -- the constructor arguments were previously ignored, which looks unintended.